suricata.yaml.in

```diff
@@ -1672,6 +1672,11 @@ decoder:
   # maximum number of decoder layers for a packet
   # max-layers: 16
 
+  # IP-in-IP tunneling for ipv4 over ipv4 handling
+  # disabled by default
+  # ipv4:
```

Comment:
I'm not sure about this key name. I know the docstring above clarifies it but `decoder.ipv4` sounds like we're talking about ipv4 packets in general. wdyt?